website/docs/networking/vpc-lattice/cleanup.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Cleanup"
+sidebar_position: 25
+---
+
+Delete VPC Lattice resources:
+
+```bash
+$ kubectl delete -f ./workspace/modules/networking/vpc-lattice/routes
+$ kubectl delete -f ./workspace/modules/networking/vpc-lattice/controller/eks-workshop-gw.yaml 
+$ kubectl delete -f ./workspace/modules/networking/vpc-lattice/controller/gatewayclass.yaml 
+```
+---
+Delete the AWS Gateway API Controller. 
+
+**THIS STEP DEPENDS FROM THE INSTALLATION METHOD YOU CHOOSE**
```

Comment:
Workshop attendees are not expected to run clean up steps manually. They should be able to simply run `reset-environment`. This page should be accurate for reference, but should be automated into the `bin/reset-environment` script.